app/models/notification.rb

```diff
@@ -70,6 +70,10 @@ def localized_message
     end
   end
 
+  def self.cancel_pending_notifications
```

Comment:
Is "pending" in the function name misleading since we're cancelling both pending and scheduled notifications?

Reply:
wdyt of
- `cancel_upcoming_notifications`
- `cancel_open_notifications`

Reply:
How about just `cancel_notifications`? Assuming the `sent` ones won't be cancelled?